client/components/faq/faq.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
```

Comment:
No worries, it's been a bit hidden away. We should probably update docs (and the i18n mixin docs) to push people into Localize. cc @aduth